tests/foreach/foreach.t

```diff
@@ -0,0 +1,95 @@
+SETUP PYTHON
+
+$ [ -n "$PYTHON" ] || PYTHON="`which python`"
+$ [ -n "$PYTHONPATH" ] || PYTHONPATH="$TESTDIR/.." && export PYTHONPATH
+
+TEST FOREACH WITH COMMAND
+
+  $ cp -r $TESTDIR/test_dir_1 test_dir_1
+  $ cd test_dir_1
+  $ git remote add origin file://$TESTDIR/test_dir_1
+  $ cd src/test_dir_2
+  $ git remote add origin file://$TESTDIR/test_dir_1/src/test_dir_2
+  $ cd ../..
+  $ cd src/test_dir_3
+  $ git remote add origin file://$TESTDIR/test_dir_1/src/test_dir_3
+  $ cd ../..
+  $ cd src/test_dir_4
+  $ git remote add origin file://$TESTDIR/test_dir_1/src/test_dir_4
+  $ cd ../..
+  $ quark freeze > /dev/null
+  $ quark foreach 'echo $name $sm_path $displaypath $sha1 $toplevel $rev $version_control'
+  .* (re)
+  .* (re)
+  test_dir_2 src/test_dir_2 src/test_dir_2 bbad4b8fc3c58b39e14956bc81d73b756a08160e /tmp/cramtests-.*/foreach.t/test_dir_1 git (re)
+   (re)
+  test_dir_3 src/test_dir_3 src/test_dir_3 f7e94048b2b2d5a3987ec00ed048fc3fac7a74f0 /tmp/cramtests-.*/foreach.t/test_dir_1 git (re)
+   (re)
+  test_dir_4 src/test_dir_4 src/test_dir_4 13a506afbbae847c8878c04f6f305dff597e3d7d /tmp/cramtests-.*/foreach.t/test_dir_1 git (re)
+   (re)
+
+
+TEST FOREACH WITH SCRIPT
+
+  $ cp -r $TESTDIR/test_dir_1 test_dir_1
+  $ cd test_dir_1
+  $ git remote add origin file://$TESTDIR/test_dir_1
+  $ cd src/test_dir_2
+  $ git remote add origin file://$TESTDIR/test_dir_1/src/test_dir_2
+  $ cd ../..
+  $ cd src/test_dir_3
+  $ git remote add origin file://$TESTDIR/test_dir_1/src/test_dir_3
+  $ cd ../..
+  $ cd src/test_dir_4
+  $ git remote add origin file://$TESTDIR/test_dir_1/src/test_dir_4
+  $ cd ../..
+  $ quark freeze > /dev/null
+  $ quark foreach $TESTDIR/foreach.sh
+  .* (re)
+  .* (re)
+  name: test_dir_2
+  sm_path: src/test_dir_2
+  displaypath: src/test_dir_2
+  version_control: git
+  sha1: bbad4b8fc3c58b39e14956bc81d73b756a08160e
+  rev: \w* (re)
+  toplevel: /tmp/cramtests-.*/foreach.t/test_dir_1/test_dir_1 (re)
+   (re)
+  name: test_dir_3
+  sm_path: src/test_dir_3
+  displaypath: src/test_dir_3
+  version_control: git
+  sha1: f7e94048b2b2d5a3987ec00ed048fc3fac7a74f0
+  rev: \w* (re)
+  toplevel: /tmp/cramtests-.*/foreach.t/test_dir_1/test_dir_1 (re)
+   (re)
+  name: test_dir_4
+  sm_path: src/test_dir_4
+  displaypath: src/test_dir_4
+  version_control: git
+  sha1: 13a506afbbae847c8878c04f6f305dff597e3d7d
+  rev: \w* (re)
+  toplevel: /tmp/cramtests-.*/foreach.t/test_dir_1/test_dir_1 (re)
+   (re)
+
+
+HELPER
```

Comment:
As discussed (I'm replicating this here just to keep track of advancement), this should be removed completely, or reduced to check just if somewhere in the output there's something that says `quark foreach`. This test, as it is, is only a burden, as it forces any modification to help messages to be replicated in two places, possibly breaking due to irrelevant differences in argparse

Reply:
Removed in https://github.com/comelz/quark/pull/27/commits/bab0b9dcbd9df472711402e149ae091f0cfa1f18